mslib/mscolab/server.py

```diff
@@ -209,13 +216,19 @@ def home():
 def hello():
     return "Mscolab server"
 
+@APP.route("/status_auth")
```

Comment:
we can do this with the origin, don't need a second one.

Reply:
In this way, older clients can still use the server. I think this is a small enough thing to retain backwards-compatibility?

Reply:
we have new client -> old server, which also does not work. 

I also thought on deprecating the /status check at all. When someone wants to remove the name, currently the service is not working for clients.

Reply:
this is not a good solution, the next one will add a third option.  And when I want to harden the server I don't want to tell something there. And someone not having this urlpath may show that some modules are not on a recent version.

So people have to update.